packages/itwinui-css/src/alert/alert.scss

```diff
@@ -16,28 +16,30 @@
   background-color: var(--iui-color-background);
   border: 1px solid var(--_iui-alert-border-color);
   box-shadow: inset var(--iui-size-2xs) 0 0 var(--_iui-alert-border-color);
-  @media (forced-colors: active) {
-    background-color: Canvas;
-  }
+}
 
-  &.iui-sticky {
-    border-radius: 0;
-    position: sticky;
-    top: 0;
-    left: 0;
-  }
+@mixin iui-alert-sticky {
+  border-radius: 0;
+  position: sticky;
+  top: 0;
+  left: 0;
 }
 
 @mixin iui-alert-icon {
   @include iui-icon-style('m');
   margin-left: var(--iui-size-m);
+  fill: var(--_iui-alert-icon-color, var(--iui-color-icon-muted));
+  @media (forced-colors: active) {
+    fill: CanvasText;
+  }
 }
 
 @mixin iui-alert-message {
   margin: var(--iui-size-s) var(--iui-size-m);
 }
 
 @mixin iui-alert-link {
+  @include iui-anchor-status;
```

Comment:
I think this may not be needed. Because all the six examples in alert.html had a variant provided. And when a variant is provided, its styling is handled in [L72-74](https://github.com/iTwin/iTwinUI/blob/28421289ed0fa9f3dedd3549a5a4275f8098f01f/packages/itwinui-css/src/alert/alert.scss#L72-L74).

Also, removing this line still passed the tests.

Reply:
Removed.